### ISSUE TITLE
System Error (No Code Execution)

### ISSUE THREAD

**Issue by tannerthomas20** [Kitware/paraviewweb#506]
Hi!

I have been trying to install Paraview on my computer for some time now. So far I can get Paraview installed with no problem. However, when I try to run the program I get the reoccurring error

"The code execution cannot proceed because msmpi.dll was not found. Reinstalling the program may fix this problem."

 (Spoiler, reinstalling the program does not fix the problem)

I have tried downloading the .exe and the .zip file and I get the same error. I also cannot find the aforementioned file in the bin folder of the .zip file. What am I missing about this installation?

Thanks!

-Tanner

Windows 10 Enterprise
Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz 2.11 GHz
8 GB
64-bit Operating System, x64 based processor

**Comment by jourdain** [comment]
You may want to report your issue [here](https://discourse.paraview.org/) while also explaining more what you did? (version of PV, command line or double clicked on the paraview.exe).

If I'm not mistaken, there is no relationship with ParaViewWeb?

**Comment by aronhelser** [comment]
Yes, this is a ParaView problem. There are two versions, "MPI" and "No MPI"
- you want the "No MPI" version.

**Comment by tannerthomas20** [comment]
Ok good to know. My intention is to run paraview in parallel with FreeCAD (i.e. be able to open an analysis from FreeCAD in ParaView). Is this still possible with the No MPI version?

Thanks

-Tanner

[QNX_EmailSigLogo_151x50]

Tanner Thomas
R & D Chemist II
Quanex IG Systems
800 Cochran Avenue
Cambridge, Ohio 43725
www.Quanex.com<http://www.quanex.com/>

**Comment by aronhelser** [comment]
I have no experience with FreeCAD, but you will be able to run other apps at the same time as ParaView with either version.
